lib/datadog/tracing/contrib/graphql/unified_trace.rb

```diff
@@ -47,14 +47,26 @@ def execute_multiplex(*args, multiplex:, **kwargs)
           end
 
           def execute_query(*args, query:, **kwargs)
-            trace(proc { super }, 'execute', query.selected_operation_name, query: query) do |span|
-              span.set_tag('graphql.source', query.query_string)
-              span.set_tag('graphql.operation.type', query.selected_operation.operation_type)
-              span.set_tag('graphql.operation.name', query.selected_operation_name) if query.selected_operation_name
-              query.variables.instance_variable_get(:@storage).each do |key, value|
-                span.set_tag("graphql.variables.#{key}", value)
-              end
-            end
+            trace(
```

Comment:
Nitpick: Can we extract those callable lambda and procs? 

I assume they are static and they looked weird to me that we are creating new ones in the memory for every invocation.

Reply:
I tried to extract them, but they fail with `undefined local variable or method 'query'`.
I forgot that they are closures that capture the value of `query` from the parent's context in order to run, so I can't extract them easily.